### PR TITLE
Create SECURITY.md

### PR DESCRIPTION
**Pull request by adiroiban**
This is to have a "Standard" security document.

Just link it to Twisted.

### CODE REVIEW

**Comment by adiroiban** [comment]
needs-review

File: SECURITY.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+# Security Policy
+
+twisted/incremental project uses the same security policy as [twisted/twisted](https://github.com/twisted/twisted).
+
+For more details please check the [Twisted securty process](https://github.com/twisted/twisted?tab=security-ov-file#readme)
```
Comment (twm) [inline SECURITY.md]:
```suggestion
The twisted/incremental project uses the same security policy as the [twisted/twisted](https://github.com/twisted/twisted) project.

For more details please refer to the [Twisted security process](https://github.com/twisted/twisted?tab=security-ov-file#readme).
```

**Review by twm — APPROVED** [review]
Minor grammar suggestions, but :shipit:! Thanks Adi.

**Comment by adiroiban** [comment]
Thanks for the review!

With this merge, there is now a dedicated link on GitHub main repo page https://github.com/twisted/incremental?tab=security-ov-file#readme

**Pull request merged**